vnext/include/ReactUWP/Views/FrameworkElementViewManager.h

```diff
@@ -20,6 +20,8 @@ class REACTWINDOWS_EXPORT FrameworkElementViewManager : public ViewManagerBase
   virtual void TransferProperties(XamlView oldView, XamlView newView) override;
 
   void TransferProperty(XamlView oldView, XamlView newView, winrt::Windows::UI::Xaml::DependencyProperty dp);
+
+  void AnnounceIfNeeded(winrt::FrameworkElement element);
```

Comment:
This is pretty nit picky, but I'd make the function name more indicative of it's tie to LiveRegionChanged.  At first blush it sounds like it might call UiaRaiseNotificationEvent which I think is a newer more general purpose announcement API.

Reply:
I'm going to move the whole thing into an accessibility helpers method rather than polluting the viewmanager since I think I'll be needing more as time goes on.